src/__tests__/purifyHTML.test.ts

```diff
@@ -0,0 +1,23 @@
+import { purifyHTMLString } from '@/utils/purifyHTML';
+import { describe, expect, it } from 'vitest';
+
+describe('purifyHTMLString', () => {
+  it('returns the same string if it is clean', () => {
+    const cleanString = 'Hello, world!';
+    expect(purifyHTMLString(cleanString)).toBe(cleanString);
```

Comment:
Follow AAA (Arrange Act Assert) convention

Reply:
Not very familiar with the AAA convention, but I will study it and implement